pipeline_tools/shared/submission/create_file_descriptor.py

```diff
@@ -82,8 +89,11 @@ def main():
 
     schema_url = args.schema_url.strip('/')
 
-    descriptor_entity_id = get_uuid5(args.sha256)
+    descriptor_entity_id = get_uuid5(
```

Comment:
I don't think this was my understanding of what was discussed with Hannes last week, but if this is what you guys landed on  it works for me